docs/en/get_started/quick_run.md

```diff
@@ -1,84 +1,167 @@
 # Quick Run
 
-In this guide we will show you some useful commands and familiarize you with MMOCR. We also provide [a notebook](https://github.com/open-mmlab/mmocr/blob/main/demo/MMOCR_Tutorial.ipynb) that can help you get the most out of MMOCR.
+## Inference
 
-## Installation
+In addition to using our well-provided pre-trained models, you can also train models on your own datasets. In the next section, we will take you through the basic functions of MMOCR by training DBNet on the mini [ICDAR 2015](https://rrc.cvc.uab.es/?ch=4&com=downloads) dataset as an example.
 
-Check out our [installation guide](install.md) for full steps.
+The next sections assume that you are using the editorial approach to install the MMOCR codebase.
 
-## Dataset Preparation
+## Prepare a Dataset
 
-MMOCR supports numerous datasets which are classified by the type of their corresponding tasks. You may find their preparation steps in these sections: [Detection Datasets](datasets/det.md), [Recognition Datasets](datasets/recog.md), [KIE Datasets](datasets/kie.md) and [NER Datasets](datasets/ner.md).
+Since the variety of OCR dataset formats are not conducive to either switching or joint training of multiple datasets, MMOCR proposes a uniform data format, and provides conversion scripts and tutorials for all commonly used OCR datasets (see the **Datasets** section). Usually, to use those datasets in MMOCR, you just need to follow the steps to get them ready for use.
 
-## Inference with Pretrained Models
+```{note}
+But here, efficiency means everything.
+```
 
-You can perform end-to-end OCR on our demo image with one simple line of command:
+Here, we have prepared a lite version of ICDAR 2015 dataset for demonstration purposes. Download our pre-prepared [zip](https://download.openmmlab.com/mmocr/data/icdar2015/mini_icdar2015.tar.gz) and extract it to the `data/det/` directory under mmocr to get our prepared image and annotation file.
 
-```shell
-python mmocr/utils/ocr.py demo/demo_text_ocr.jpg --print-result --imshow
+```Bash
+wget https://download.openmmlab.com/mmocr/data/icdar2015/mini_icdar2015.tar.gz
+mkdir -p data/det/
+tar xzvf mini_icdar2015.tar.gz -C data/det/
 ```
 
-Its detection result will be printed out and a new window will pop up with result visualization. More demo and full instructions can be found in [Demo](demo.md).
+## Modify the Config
 
-## Training
+Once the dataset is prepared, we will then specify the location of the training set and the training parameters by modifying the config file.
 
-### Training with Toy Dataset
+In this example, we will train a DBNet using resnet18 as its backbone. Since MMOCR already has a config file for the full ICDAR 2015 dataset (`configs/textdet/dbnet/dbnet_resnet18_fpnc_1200e_ icdar2015.py`), we just need to make some modifications on top of it.
```

Comment:
```suggestion
In this example, we will train a DBNet using resnet18 as its backbone. Since MMOCR already has a config file for the full ICDAR 2015 dataset (`configs/textdet/dbnet/dbnet_resnet18_fpnc_1200e_icdar2015.py`), we just need to make some modifications on top of it.
```